src/toil/batchSystems/slurm.py

```diff
@@ -183,9 +183,13 @@ def prepareSbatch(self, cpu, mem, jobID):
         sbatch_line = ['sbatch', '-Q', '-J', 'toil_job_{}'.format(jobID)]
 
         if self.boss.environment:
+            argList = []
+            
             for k, v in self.boss.environment.iteritems():
```

Comment:
I believe the python3 support changes from https://github.com/BD2KGenomics/toil/pull/1380 are what are causing the conflicts between your branch and master. I think if you merge the master branch into this PR and change this line to use the version of `iteritems` imported from `six`:
   
    for k, v in iteritems(self.boss.environment):

we can resolve the conflicts and merge this!